tests/components/observers/test_household_survey_observers.py

```diff
@@ -51,40 +62,50 @@ def test_on_simulation_end(observer, mocker):
     ).is_file()
 
 
-def test_do_observation(observer, mocked_pop_view, mocker):
+def test_get_observation(
+    observer, mocked_pop_view, mocked_household_details_pipeline, mocker
+):
     """Are responses recorded correctly (including concatenation after time steps?"""
     sim_start_date = "2021-01-01"
     # Simulate first time step
     event = mocker.MagicMock()
     event.time = pd.to_datetime(sim_start_date)
     event.index = pd.Index([0, 1])
-    observer.population_view.get.return_value = mocked_pop_view  # FIXME: This returns mocked_pop_view regardless of event.index or alive status
+    # FIXME: This returns mocked_pop_view regardless of event.index or alive status
+    observer.population_view.get.return_value = mocked_pop_view
```

Comment:
Does this mean it is returning all of `mocked_pop_view` no matter what?  If I understand the mocks correctly I think you are just assigning it here.  You need to mock population_view in the mocked observer's builder.

Reply:
You can do it either way. If we wanted to be able to get variable results as that comment suggests, this approach won't enable it. I'm going to consider that out of scope for this PR ... it's big enough as it is.

Reply:
Ahh ok that makes sense.